src/style.css

```diff
@@ -1 +1,72 @@
 /* Styles go here */
+* {
+  max-width: 1200px;
```

Comment:
you definitely don't want the `max-width` prop. to be applied to everything. HTML-wise, usually you create a `div` container inside your `main`, which contains all the code and which only purpose is to limit the maximum width.
but for now you can just move it to the `.header`